proto/palomachain/paloma/evm/tx.proto

```diff
@@ -39,3 +35,45 @@ message MsgRemoveSmartContractDeploymentRequest {
       [ (gogoproto.nullable) = false ];
 }
 message RemoveSmartContractDeploymentResponse {}
+
+message MsgUploadUserSmartContractRequest {
+  option (cosmos.msg.v1.signer) = "metadata";
+
+  palomachain.paloma.valset.MsgMetadata metadata = 1
+      [ (gogoproto.nullable) = false ];
+
+  string val_address = 2 [ (cosmos_proto.scalar) = "cosmos.AddressString" ];
```

Comment:
Probably not needed as part of the request. We can extract the value from the Metadata, right?